.travis.yml

```diff
@@ -48,4 +53,4 @@ script:
   # Run bioblend nosetests with the same UID and GID as the galaxy user inside if Docker
   # this will guarantee that exchanged files bewteen bioblend and Docker are read & writable from both sides
   - sudo -E su $GALAXY_TRAVIS_USER -c "export PATH=$GALAXY_HOME/.local/bin/:$PATH && cd $GALAXY_HOME/bioblend-0.7.0 && tox -e $TOX_ENV"
-  - docker run galaxy-docker/test  bash -c 'install-repository "--url https://toolshed.g2.bx.psu.edu -o iuc --name bedtools --panel-section-name 'BEDTools'"'
```

Comment:
Can we leave both installation tests for a few releases. I'm fine with changing the default installation method to the new script, but we should offer and test the old one as well, imho.